app/poros/item.rb

```diff
@@ -0,0 +1,14 @@
+class Item
+  attr_reader :id,
+              :name,
+              :description,
+              :unit_price
+
+  def initialize(data)
+    @id = data[:id]
+    @name = data[:attributes][:name]
+    @description = data[:attributes][:description]
+    @unit_price = data[:attributes][:unit_price]
+    @merchant_id = data[:attributes][:merchant_id]
+  end
+end
```

Comment:
THis goes for the merchant class too, but be careful naming an attribute "id", since poros automatically have a unique id assocoiated with them. THis could cause some issues when you're trying to access the id that you manually set later on.